lib/templates/template.tex

```diff
@@ -1,5 +1,8 @@
 \documentclass[$if(fontsize)$$fontsize$$else$12pt$endif$,$if(lang)$$lang$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
 
+\providecommand{\tightlist}{%
```

Comment:
What I'll do is update this PR so...

- the `\tightlist` stuff is in the same location in the file as the pandoc default template
- add a comment to the top to say this this the Underscore modified version of the default template